components/azure_sql/actions/execute-raw-query/execute-raw-query.mjs

```diff
@@ -0,0 +1,31 @@
+import app from "../../azure_sql.app.mjs";
+
+export default {
+  key: "azure_sql-execute-raw-query",
+  name: "Execute SQL Query",
+  description: "Execute a custom SQL query. See [our docs](https://pipedream.com/docs/databases/working-with-sql) to learn more about working with SQL in Pipedream.",
+  type: "action",
+  version: "0.0.1",
+  props: {
+    app,
+    // eslint-disable-next-line pipedream/props-description
+    sql: {
+      type: "sql",
+      auth: {
+        app: "azure_sql",
```

Comment:
I think this needs to be `app: "app"` since you imported `app` from azure_sql.app.mjs?

Reply:
That worked, thanks @danhsiung! I had mistakenly assumed that was the app slug.